t3f/ops.py

```diff
@@ -747,16 +750,28 @@ def multiply(tt_left, right):
   is_right_batch = isinstance(right, TensorTrainBatch)
 
   is_batch_case = is_left_batch or is_right_batch
-
+  ndims = tt_left.ndims()
   if not isinstance(right, TensorTrainBase):
     # Assume right is a number, not TensorTrain.
     tt_cores = list(tt_left.tt_cores)
-    tt_cores[0] = right * tt_cores[0]
+    if not fast:
```

Comment:
Sorry, after all lets not do this flag and just always be robust and a bit slow. Can you please revert the flag part?